isaric/parsers/idor-brazil.toml

```diff
@@ -0,0 +1,760 @@
+
+[subject.pregnancy]
+  field = "pregyn_rptestcd"
+  description = "Is the patient pregnant?"
+  values = { 1 = true, 0 = false }
+
+[subject.pregnancy_date_of_delivery]
+  field = "dlvrdtc_rptestcd"
+  description = "Delivery date"
+
+[subject.pregnancy_gestational_age_weeks]
+  field = "egestage_rptestcd"
+  description = "If yes: gestational age (in week of amenorrhea)"
+  unit = "weeks"
+
+[subject.pregnancy_birth_weight_kg]
+  field = "apvs_weight"
+  unit = "kg"
+  source_unit = { field = "apvs_weightu", values = { 1 = "kg", 2 = "lb"}}
+
+[subject.pregnancy_gestational_outcome]
+  field = "apsc_gestout"
+  description = "Pregnancy gestational outcome"
+  values = { 1 = "term_birth", 2 = "preterm_birth" }
+
+[subject.pregnancy_whether_breastfed]
+  field = "apsc_brfedind"
+  ref = "Y/N/NK"
+
+[subject.pregnancy_outcome]
+  field = "pregout_rptestcd"
+  description = "Pregnancy"
+  values = { 1 = "live birth", 2 = "still birth" }
+
+[subject.pregnancy_post_partum]
+  field = "postpart_rptestcd"
+  description = "Post partum?"
+  values = { 1 = true, 0 = false }
+
+[subject.icu_admitted]
+  field = "icu_hoterm"
+  ref = "Y/N/NK"
+
+[subject.sex_at_birth]
+  field = "sex"
+  description = "Sex"
+  values = { 1 = "male", 2 = "female" }
+
+[subject.works_healthcare]
+  field = "healthwork_erterm"
+  ref = "Y/N/NK"
+
+[subject.works_microbiology_lab]
+  field = "labwork_erterm"
+  ref = "Y/N/NK"
+
+[subject.has_comorbidity_other]
+  combinedType = "set"
+  excludeWhen = "none"
+  fields = [{ field = "otherrisktext", if.other_mhyn = 1 }]
+
+  # visit
+
+[visit]
+  country_iso3 = "BRA"
+
+  [visit.subject_id]
+    field = "subjid"
+    sensitive = true
+
+  [visit.visit_id]
+    field = "subjid"
+    sensitive = true
+
+  [visit.start_date]
+    ref = "admissionDate"
+
+  [visit.icu_admission]
+    field = "daily_hoterm" # should this use icu_hoterm as well?
+    ref = "Y/N/NK"
+
+  [visit.icu_admission_dates]
+    combinedType = "set"
+    excludeWhen = "none"
+    fields = [ { field = "icu_hostdat"}]
+
+  [visit.transfer_from_other_facility]
+    field = "hooccur"
+    values = { 1 = true, 2 = true, 3 = false }
+
+  [visit.treatment_prone_position]
+    combinedType = "any"
+    fields = [
+      { field = "pronevent_prtrt", ref = "Y/N/NK" },
+      { field = "daily_prone_cmtrt", ref = "Y/N/NK" },
+    ]
+
+  [visit.outcome]
+    field = "dsterm"
+    description = "Outcome"
+
+    [visit.outcome.values] # TODO: check
+      1 = "discharged"
+      2 = "hospitalised"
+      3 = "transferred"
+      4 = "death"
+      5 = "palliative discharge"
+
+[visit.date_outcome]
+  field = "dsstdtc"
+  description = "Outcome date"
+
+[visit.treatment_dialysis]
+  combinedType = "any"  # there is also renal_proccur in outcome (desfecho)?
+  fields = [
+    { field = "rrt_prtrt", ref = "Y/N/NK" },
+    { field = "daily_rrt_cmtrt", ref = "Y/N/NK" },
+  ]
+
+[visit.pathogen_test_date] # TODO: check
+  field = "mbdat"
```

Comment:
If mbtestcd = SARS COV 2 / covid19 ? (it's free text unfortunately with a fair number of spelling variants) - lots of other testing was also done

Reply:
Added matches using the new =~ operator